airgun/entities/hostcollection.py

```diff
@@ -172,6 +176,33 @@ def install_errata(self, entity_name, errata_id,
             )
             return job_status_view.overview.read()
 
+    def manage_module_streams(self, entity_name, action_type,
+                              module_name, stream_version):
```

Comment:
Same as in `entities/contenthost.py`: does it make sense to provide some default value to some of these arguments, from product point of view?